docs/react-native/docs/ios/remote-notification-support.md

```diff
@@ -100,4 +101,4 @@ Now everything is setup in your app, you can alter your notification payload to
 
 Please note, the `id` of the notification is the `request.identifier` and cannot be changed. For this reason, the `id` property in `notifee_options` should be excluded.
 
-> if both `attachments` and `image` are present, `attachments` will take precedence over `image`
\ No newline at end of file
+> if both `attachments` and `image` are present, `attachments` will take precedence over `image`
```

Comment:
This is just a curiosity - I cannot see any actual diff on this line, I'm experimenting (with this suggestion) with copy-pasting the original in place of the current to see if it quiets the diff

```suggestion
> if both `attachments` and `image` are present, `attachments` will take precedence over `image`
```

Reply:
nope - deleting both.